src/components/MapSelect/index.js

```diff
@@ -131,7 +131,7 @@ const MapSelect = ({
                 feature.properties[iconField],
                 selection.current.has(feature.properties[idField])
               ),
-              alt: feature.properties.objectnummer,
+              alt: selectionOnly ? '' : feature.properties.objectnummer,
```

Comment:
doesn't this give you an empty property? `alt=""` or does it not set it with an empty string?

Reply:
Yes it is set to `alt=""`. This is a valid method to prevent the screenreader from reading out lout other info contained in the tag.